### ISSUE TITLE
Bundle renovate

### ISSUE THREAD

**Issue by viceice** [renovatebot/docker-renovate#26]
bundle renovate so we don't need full node_modules folder

**Comment by rarkins** [comment]
@viceice this is no longer blocked by v20, right?

**Comment by viceice** [comment]
Yes, I need some time to think about it. 

It needs tests, so we don't break silently.

**Comment by viceice** [comment]
Because of #34 we can no longer use `@zeit/ncc` because it can only generate a single output, but we need two.
We need `renovate` and `config-validator` and we don't want two big files which would contradict bundling.

🤔 Or should we simply ignore the duplicated stuff

**Comment by viceice** [comment]
Won't do it here
